### PR TITLE
Write required disk size to update manifest

### PR DESCRIPTION
**Pull request by themylogin**

### CODE REVIEW

File: scripts/build_manifest.py [hunk 1]
```diff
@@ -29,5 +29,7 @@
             "date": datetime.utcnow().isoformat(),
             "version": version,
             "checksums": checksums,
+            # Unpacked root squashfs consumes on average twice its size on ZFS dataset + 10% just in case
+            "size": int(os.path.getsize(os.path.join("output", "rootfs.squashfs")) * 2 * 1.1),
```
Comment (william-gr) [inline scripts/build_manifest.py:33]: Should the size just reflect reality and we put the 2 * 1.1 in the code?

Comment (themylogin) [reply scripts/build_manifest.py:33]: I prefer to leave more control in our (update file providers) hands, e.g. we can calculate this value precisely if we ever need this.

Comment (william-gr) [reply scripts/build_manifest.py:33]: I dont understand, how that does not leave the control in our hands? The size is a property in the manifest, why purpose does it serve the manfiest to lie about the exact size of a file we ship?

Comment (themylogin) [reply scripts/build_manifest.py:33]: We say "to install this, you'll need at least `size` space available in your boot pool"

Comment (william-gr) [reply scripts/build_manifest.py:33]: Thats a guess at best. I think we need to either save the real size it uses before squashing it if thats what we are after.

**Pull request merged**